### DISCUSSION TITLE
ActiveMQ Artemis

### DISCUSSION THREAD

**Discussion by goschtl** [celery/kombu#1433] [Q&A]
Hi,

i try to use kombu in an ActiveMQ / Artemis Environment. It seems there are many ways to establish a connection.
Unfortunatly none of these seems to work.

- **qpid** support is only for python2.7
- **kombu-stomp** is unmaintained and old
- **amqp** ActiveMQ Artemis uses a AMQP 1.0 Version

Does anyone have an Idea how to deal with it?

Thanks in advance
Christian

**Comment by auvipy** [comment]
you should try qpid-proton instead of kombu IMHO as kombu don't support amqp 1.0 yet

**Reply by goschtl** [reply]
Hey thanks for your quick answer. Are there any plans in the future to get amqp 1.0 in kombu and any plan for a qpid-proton transport?

**Comment by auvipy** [comment]
yes
